src/dispatch/plugins/dispatch_slack/case/messages.py

```diff
@@ -345,11 +347,13 @@ def create_genai_signal_summary(
     genai_plugin = plugin_service.get_active_instance(
         db_session=db_session, project_id=case.project.id, plugin_type="artificial-intelligence"
     )
-    if not genai_plugin:
+
+    print(f"{signal_instance.signal.__dict__}")
+    if not genai_plugin or not signal_instance.signal.genai_prompt:
         return signal_metadata_blocks
```

Comment:
Should we separate the checks and log a warning when `genai_plugin` exists, but we don't have a prompt?

Reply:
Good idea, done